src/cli/main.cpp

```diff
@@ -58,21 +58,38 @@ int main(int argc, char* argv[]) {
   const std::string programName = std::move(args.front());
   args.erase(args.begin());
 
-  const bool is_labs = !args.empty() && (args[0] == "labs");
-  if (is_labs) {
+  const bool is_classic = !args.empty() && (args[0] == "classic");
+
+  if (is_classic) {
+    // The "classic" cli implementation will not be expecting the first arg
+    // to be the word classic so we need to remove the first arg
+    args.erase(args.begin());
+  } else {
     // Replace backward slashes with forward slashes... cf #4856
-    std::for_each(args.begin(), args.end(), [](auto& s) {
-      //std::replace(s.begin(), s.end(), '\\', '/');
-      boost::replace_all(s, "\\", "\\\\");
-    });
-    // fmt::print("Cleaned after slash replacement arguments: {}\n", args);
+    std::for_each(args.begin(), args.end(), [](auto& s) { boost::replace_all(s, "\\", "\\\\"); });
   }
 
   // ScriptEngineInstance will delay load the engines
   openstudio::ScriptEngineInstance rubyEngine("rubyengine", args);
   openstudio::ScriptEngineInstance pythonEngine("pythonengine", args);
 
-  if (is_labs) {
+  if (is_classic) {
+    fmt::print(fmt::fg(fmt::color::orange),
+               "┌{0:─^{2}}┐\n"
+               "│{1: ^{2}}│\n"
+               "└{0:─^{2}}┘",
+               "", "The `classic` command is deprecated and will be removed in a future release", 80);
+    fmt::print("\n");
```

Comment:
@kbenne I added a banner to the "classic" subcommand, maybe you disagree with that part though? This can be reverted.

Reply:
![image](https://github.com/NREL/OpenStudio/assets/5479063/2be820ab-53a4-43eb-96fa-4b94bf23ca68)